.changeset/tiny-cobras-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': minor
+---
+
+Expose an ``onAuth` handler for `git` actions to provide custom credentials
```

Comment:
Expose an `onAuth` handler for `git` actions to provide custom credentials

Reply:
Removed the extra backtick